packages/web-components/src/components/va-file-input/va-file-input.css

```diff
@@ -0,0 +1,24 @@
+@import '../../mixins/accessibility.css';
+@import '../../mixins/form-field-error.css';
+
+:host {
+  display: block;
+  font-family: var(--font-source-sans);
+}
+
+#error-message{
+  margin-bottom: 0.3rem;
+}
+
+va-button {
+  margin-bottom: -0.8rem;
+}
+
+.required {
+  color: var(--color-secondary-dark);
+  margin-left: 0.4rem;
+}
+
+.hidden {
+  display: none;
+}
```

Comment:
Would adding the [`hidden` attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/hidden) to the `<input>` element do the same thing?

Reply:
I was concerned that `hidden` would affect screen readers in some way although now that I'm testing both ways, I can't find a functional difference so I think that's a good callout and streamlines it a bit more.

Reply:
Resolved here: https://github.com/department-of-veterans-affairs/component-library/pull/507/commits/89adf95016f7a8b180d8cda20c7743e31b611206